src/mixins.ts

```diff
@@ -38,7 +38,7 @@ export default {
       return shorten(str);
     },
     _ipfsUrl(ipfsHash: string): string | null {
-      if(!ipfsHash) return null;
+      if (!ipfsHash) return null;
```

Comment:
I've seen this happen a lot in other PR's, any idea where the syntax without a space is coming from?

Reply:
not sure about others but I got it when I run lint 🤔 can remove it?

Reply:
No, I also have the syntax with the space. Was just wondering why this space keeps getting removed.